app/models/ability.rb

```diff
@@ -70,7 +70,7 @@ def initialize(user)
       (user_is_admin_of?(group.id) && group.enabled_beta_features.include?('export'))
     end
 
-    can [:members_autocomplete, :set_volume, :see_members, :move_discussions_to, :view_previous_proposals], Group do |group|
+    can [:members_autocomplete, :set_volume, :add_discussion, :see_members, :move_discussions_to, :view_previous_proposals], Group do |group|
```

Comment:
Yeah, the ability is 'given a group, can I make a discussion in it'?
Alternative might be do do something like `can? :create,
group.discussions.build`

On Tue, Nov 3, 2015 at 11:32 PM, Robert Guthrie <notifications@github.com>
wrote:

> In app/models/ability.rb
> <https://github.com/loomio/loomio/pull/2683#discussion_r43810079>:
>
> > @@ -70,7 +70,7 @@ def initialize(user)
> >        (user_is_admin_of?(group.id) && group.enabled_beta_features.include?('export'))
> >      end
> >
> > -    can [:members_autocomplete, :set_volume, :see_members, :move_discussions_to, :view_previous_proposals], Group do |group|
> > +    can [:members_autocomplete, :set_volume, :add_discussion, :see_members, :move_discussions_to, :view_previous_proposals], Group do |group|
>
> these add_ methods are different to :create, discussion because you don't
> pass the discussion object eh?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/2683/files#r43810079>.
>